### PR TITLE
Filter to CSC data

### PR DESCRIPTION
**Pull request by Br3nda**
# What has changed and why?
filtering to results that mention community services card

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

**Comment by Br3nda** [comment]
@dlouise64 this one still waiting for your review

**Comment by dlouise64** [comment]
@Br3nda Let me know when this one is ready to review

File: src/actions/index.js [hunk 1]
```diff
@@ -60,8 +69,10 @@ export function loadService(searchVars,serviceId){
 }
 
 function requestBuilder(searchVars){
-  let theq = (searchVars.keyword && searchVars.keyword.length > 2) ? '&q='+searchVars.keyword : '';
+  let q = (searchVars.keyword && searchVars.keyword.length > 2) ? searchVars.keyword : '';
+  let theq = `&q=${q} ${GLOBAL_FILTER}`;
   let url = encodeURI(`${API_PATH}datastore_search?resource_id=${RESOURCE_ID}&fields=${STATICFIELDS}${theq}&distinct=true${filters(searchVars.category)}`);
+  console.log(url);
```
Comment (dlouise64) [inline src/actions/index.js]: @Br3nda remove console logging

File: src/actions/index.js [hunk 2]
```diff
@@ -3,12 +3,21 @@ import geolib from 'geolib';
...
 export function loadFilters(){
-  let sql = encodeURI(`SELECT "LEVEL_1_CATEGORY" as name, COUNT(*) as num FROM "${RESOURCE_ID}" GROUP BY name ORDER BY name`);
+  let fields = '"LEVEL_1_CATEGORY" as name, COUNT(*) as num';
+  let where = `WHERE "SERVICE_DETAIL" LIKE \'%${GLOBAL_FILTER}%\'
+    OR "SERVICE_TARGET_AUDIENCES" LIKE \'%${GLOBAL_FILTER}%\'
+    OR "COST_DESCRIPTION" LIKE \'%${GLOBAL_FILTER}%\'
+    OR "DELIVERY_METHODS" LIKE \'%${GLOBAL_FILTER}%\'`;
+
+  let sql =`SELECT ${fields} FROM "${RESOURCE_ID}" ${where} GROUP BY name ORDER BY name`;
+  console.log(sql);
```
Comment (dlouise64) [inline src/actions/index.js]: @Br3nda can remove this console log as well

File: src/actions/index.js [hunk 3]
```diff
@@ -3,12 +3,21 @@ import geolib from 'geolib';
 
 const RESOURCE_ID = process.env.REACT_APP_API_RESOURCE_ID;
 const API_PATH = process.env.REACT_APP_API_PATH;
+const GLOBAL_FILTER="community services card";
```
Comment (dlouise64) [inline src/actions/index.js]: @Br3nda  just some formatting here, spaces should surround assignment operator

**Comment by Br3nda** [comment]
@dlouise64 should we still use this? the code moved on a long way. I'm gonna close and if we need it we'll re-open.

**Pull request closed without merging**